wp-content/themes/core/components/routes/not_found/css/not_found.pcss

```diff
@@ -0,0 +1,30 @@
+.not-found {
+	text-align: center;
+	margin: var(--spacer-60) 0;
+
+	@media (--viewport-medium) {
+		margin: var(--spacer-80) 0;
+	}
+}
+
+.not-found__title {
+
+}
+
+.no-found__content {
+	font-size: var(--font-size-body);
+	font-family: var(--font-family-serif);
+	max-width: var(--grid-7-col);
+	margin: var(--spacer-20) auto var(--spacer-60);
+
+	@media (--viewport-medium) {
+		margin-top: var(--spacer-30);
+		margin-bottom: 80px;
+	}
+}
```

Comment:
Can we use existing typography & grid classes for this rather than one-offs specific to this template?